napari/utils/info.py

```diff
@@ -96,6 +96,43 @@ def _napari_from_conda() -> bool:
     return any(napari_pattern.match(file.name) for file in napari_conda_files)
 
 
+def get_plugin_list() -> str:
+    """Get a list of installed plugins.
+
+    Returns
+    -------
+    str
+        A string containing the names and versions of installed plugins.
+    """
+    try:
+        from npe2 import PluginManager
+
+        pm = PluginManager.instance()
+        pm.discover(include_npe1=True)
+        pm.index_npe1_adapters()  # type: ignore[no-untyped-call]
+        fields = [
+            'name',
+            'package_metadata.version',
+            'contributions',
+        ]
+        pm_dict = pm.dict(include=set(fields))
+
+        res = []
+
+        for plugin in pm_dict['plugins'].values():
+            count_contributions = sum(
+                len(x)
+                for x in plugin['contributions'].values()
+                if x is not None
+            )
+            res.append(
+                f'  - {plugin["name"]} {plugin["package_metadata"]["version"]} ({count_contributions} contributions)'
```

Comment:
```suggestion
                f'  - {plugin["name"]}: {plugin["package_metadata"]["version"]} ({count_contributions} contributions)'
```
Nitpick to match the other entries